using-timescaledb/data-retention.md

```diff
@@ -20,6 +26,68 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention and Continuous Aggregates
+
+The function `drop_chunks` can be used to control data retention for
+the hypertable as well as the continuous aggregate. Given a hypertable
+`conditions` and a continuous aggregate `conditions_summary` over the
+hypertable:
+
+```sql
+CREATE VIEW conditions_summary WITH (timescaledb.continuous) AS
+SELECT location,
+       time_bucket('1d', "time") AS day,
+       AVG(temperature) AS avg_temperature,
+       MAX(temperature) AS max_temperature,
+       MIN(temperature) AS min_temperature
+FROM conditions
+GROUP BY location, time_bucket('1d', "time")
+```
+
+Data retention on a continous aggregate will normally not propagate to
+the hypertable, so the call below will only remove chunks from
+`conditions_summary`.
+
+```sql
+SELECT drop_chunks(INTERVAL '30 days', 'conditions_summary');
+```
+
+If you want to remove chunks from `conditions` as well, you need to
+cascade the drop chunks to the hypertable using the
+`cascade_to_materialization` option:
+
+```sql
+SELECT drop_chunks(INTERVAL '30 days', 'conditions_summary',
+                   cascade_to_materialization => TRUE);
+```
+
+It can also be useful to keep the aggregated data only but clean up
+old hypertable chunks to reduce storage cost. Since a drop of the old
+chunks will trigger an update of the continuous aggregate and remove
```

Comment:
Is it really accurate that drop_chunks on a hypertable will also remove the downsampled data (aggregate)? This doesn't sound right to me. AFAIK, it would only cascade to the aggregate if `cascade_to_materialization` is set to TRUE. (Again, this confuses me too so I am not 100% sure about this).

Reply:
This was my understanding, but maybe @gayyappan can clarify if it is accurate.

Reply:
The non-aggregated table `conditions` is not a materialization so if the drop_chunks on the aggregate that cascades to `conditions` because `cascade_to_materialization` is set doesn't make any sense.

Reply:
drop of the raw hypertable chunks cascades to aggregates ( IF cascade_to_materializations is set to true). Not vice-versa.